upd_script/fetch_sensors.sh

```diff
@@ -25,8 +25,12 @@ else
     # change_branch $BRANCH  # change to a branch we're working on, if we've defined the branch above.
 fi
 
-sudo python $SENSOR_DIR/Python/setup.py install
-sudo python3 $SENSOR_DIR/Python/setup.py install
+cd $SENSOR_DIR/Python
 
-sudo python $SENSOR_DIR/Python/di_sensors/DHT_Sensor/setup.py install
-sudo python3 $SENSOR_DIR/Python/di_sensors/DHT_Sensor/setup.py install
+sudo python setup.py install
+sudo python3 setup.py install
+
+cd $SENSOR_DIR/Python/di_sensors/DHT_Sensor
```

Comment:
The issue of not being in the same folder at the end is true depending on the `if`'s condition. That's correct.